policyengine_canada/tests/gov/cra/tax/income/credits/gst_credit/is_eldest_child_in_single_household_for_gst_credit.yaml

```diff
@@ -0,0 +1,17 @@
+- name: Base amounts for family of 3
+  period: 2023
+  input:
```

Comment:
just `person_index`, `is_child_for_gst_credit`, and `is_married` as inputs

`age` logic should be covered in the `person_index` unit test